### PR TITLE
vstart: add --help documentation for rgw_num

### PR DESCRIPTION
**Pull request by alimaredia**
Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

**Comment by alimaredia** [comment]
Should I do the same for ceph-mgr?

**Comment by liewegas** [comment]
yes please!

**Comment by tchaikov** [comment]
```
[ RUN      ] TestImageReplayer.Resync
...
Total Test time (real) = 4123.33 sec

The following tests FAILED:
	162 - unittest_rbd_mirror (Timeout)
Errors while running CTest
```
https://jenkins.ceph.com/job/ceph-pull-requests/19683/console

retest this please.

**Comment by alimaredia** [comment]
added mgr_num and just rebased. Is there anything else I need to do?

**Comment by alimaredia** [comment]
@tchaikov can we merge this soon?

File: src/vstart.sh [hunk 1]
```diff
@@ -126,9 +126,14 @@ usage=$usage"\t-X disable cephx\n"
 usage=$usage"\t--hitset <pool> <hit_set_type>: enable hitset tracking\n"
 usage=$usage"\t-e : create an erasure pool\n";
 usage=$usage"\t-o config\t\t add extra config parameters to all sections\n"
+usage=$usage"\tmon : start just ceph monitor's\n"
+usage=$usage"\tosd : start just ceph osd's\n"
+usage=$usage"\tmds : start just ceph mds's\n"
```
Comment (cbodley) [inline src/vstart.sh]: the plurals shouldn't have `'` in them (and the plural of 'mds' is 'mdses' according to @adamemerson)

File: src/vstart.sh [hunk 2]
```diff
@@ -126,9 +126,14 @@ usage=$usage"\t-X disable cephx\n"
 usage=$usage"\t--hitset <pool> <hit_set_type>: enable hitset tracking\n"
 usage=$usage"\t-e : create an erasure pool\n";
 usage=$usage"\t-o config\t\t add extra config parameters to all sections\n"
+usage=$usage"\tmon : start just ceph MONs\n"
+usage=$usage"\tosd : start just ceph OSDs\n"
+usage=$usage"\tmds : start just ceph MDSes\n"
```
Comment (tchaikov) [inline src/vstart.sh]: this is not true anymore, because mgr is always started.

Comment (tchaikov) [inline src/vstart.sh]: okay, i just pulled together a fix at #13974

**Comment by liewegas** [comment]
fwiw i'd rather remove these 'mon osd mds' arguments and have people use 
the MDS= MGR= etc env vars.

**Comment by tchaikov** [comment]
@liewegas which means if `MDS=0` no MDS will be started? and if none of `{MDS|MGR|OSD|MON}` is found in env vars, all of them will be started, right?

**Comment by alimaredia** [comment]
Yeah, I thought about adding an example of starting the daemons with env vars to the usage but didn't want to confuse new people who'll read the usage for the first time.

**Comment by liewegas** [comment]
MON/OSD/MDS default to 3, MGR to 1, RGW to 0.

**Pull request merged**